app/Eloquent/Owner.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Eloquent;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Owner extends Model
+{
+    use SoftDeletes;
+
+    protected $table = 'owners';
+
+    protected $fillable = [
+    	'book_id',
+    	'user_id',
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.DisallowTabIndent.TabsUsed</br>**Severity**: error</br>**File**: app/Eloquent/Owner.php</br>Spaces must be used to indent lines; tabs are not allowed